test/libsolidity/syntaxTests/functionCalls/eof/calloptions_on_staticcall.sol

```diff
@@ -4,7 +4,7 @@ contract C {
     }
 }
 // ====
-// EVMVersion: >=prague
+// EVMVersion: >=osaka
 // bytecodeFormat: >=EOFv1
```

Comment:
The `EVMVersion` requirement here is redundant and should be removed. The test requires an EVM version supporting EOF and `bytecodeFormat: >=EOFv1` already ensures that.

Same in other tests. I've been pointing it out in reviews, but apparently a few cases still slipped through.

Reply:
I've updated the tests in this PR, however, note that most of the other EOF tests have a `>=shanghai` constraint, which should also be removed (since all of these are superseded with `>=EOFv1` anyway). These should be updated in another PR however, since there's a ton of them.

Reply:
Looks like you did the others in the PR but left this one unchanged :)

And yeah, we can do `>=shanghai` separately. They're at least less annoying because they don't use latest EVM so probably no one will feel the need to bump them when a new EVM version is added.